exchange/src/utils/local_algos/dev_mode.cpp

```diff
@@ -0,0 +1,47 @@
+#include "dev_mode.hpp"
+
+#include "config.h"
+#include "file_management.hpp"
+#include "logging.hpp"
+
+namespace nutc {
+namespace dev_mode {
+
+void
+initialize_client_manager(manager::ClientManager& users, int num_users)
+{
+    for (int i = 0; i < num_users; i++) {
+        std::string uid = "algo_" + std::to_string(i);
+        users.add_client(uid, STARTING_CAPITAL, true);
+    }
+}
+
+void
+create_mt_algo_files(int num_users) noexcept
+{
+    std::string content = file_mgmt::read_file_content("./template.py");
+    std::string dir_name = std::string(ALGO_DIR);
```

Comment:
I'd recommend `std::filesystem::path` but this should be fine too

Reply:
This is just to convert algo_dir to a string